celery/canvas.py

```diff
@@ -319,6 +325,10 @@ def _inner(subclass):
 
     @classmethod
     def from_dict(cls, d, app=None):
```

Comment:
While you're documenting this methods, it would also be useful to annotate the types of the arguments and the return value.